busy_indicator_view.py

```diff
@@ -56,23 +56,18 @@ def on_activated_async(self, active_view):
         draw(**State)
 
 
-indicators = [
-    'Linting.  ',
-    'Linting.. ',
-    'Linting. .',
-    'Linting ..',
-    'Linting  .',
-]
+def tick():
+    while True:
+        for phase in ('.  ', '.. ', '. .' ' ..', '  .'):
+            yield "Linting" + phase
 
 
 def draw(active_view, running, **kwargs):
     buffer_id = active_view.buffer_id()
     start_time = running.get(buffer_id, None)
     now = time.time()
     if start_time and (INITIAL_DELAY <= (now - start_time) < TIMEOUT):
-        num = len(indicators)
-        text = indicators[int((now - start_time) * 1000 / CYCLE_TIME) % num]
-        active_view.set_status(STATUS_BUSY_KEY, text)
+        active_view.set_status(STATUS_BUSY_KEY, next(tick()))
```

Comment:
This doesn't work bc every time you draw you start a new generator `tick()` and take one value out of it via `next()`.